.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python: 2.7
+env:
+  - TOX_ENV=flake8
+  - TOX_ENV=py27-dj16-cms30
+  - TOX_ENV=py27-dj16-cms31
+  - TOX_ENV=py27-dj17-cms30
+  - TOX_ENV=py27-dj17-cms31
```

Comment:
Let's put the latest versions of things on top, so, should have dj17 tests before dj16 tests and cms31 tests before cms30 tests, etc.